float32 gyro_x
float32 gyro_y
float32 gyro_z
float32 accel_x
float32 accel_y
float32 accel_z
float32 magnetic_x
float32 magnetic_y
float32 magnetic_z
